arrays/arrays.c

```diff
@@ -108,14 +108,27 @@ char *arr_read(Array *arr, int index) {
 void arr_insert(Array *arr, char *element, int index) {
 
     // Throw an error if the index is greater than the current count
+    if (index >= arr->count) {
```

Comment:
`index > arr->count`  if `index == arr->count` and you insert there, you're essentially appending